core/src/main/scala/org/apache/spark/scheduler/TaskScheduler.scala

```diff
@@ -23,8 +23,7 @@ import org.apache.spark.storage.BlockManagerId
 import org.apache.spark.util.AccumulatorV2
 
 /**
- * Low-level task scheduler interface, currently implemented exclusively by
- * [[org.apache.spark.scheduler.TaskSchedulerImpl]].
```

Comment:
I would prefer keep this line, since we still have this assumption across Spark internal, and YARN scheduler is still a `TaskSchedulerImpl`.

Reply:
That makes sense. Thank you for your explanation. I'll revert this change.